zipline/utils/math_utils.py

```diff
@@ -14,7 +14,22 @@
 # limitations under the License.
 
 import math
+import numpy as np
 
 
 def tolerant_equals(a, b, atol=10e-7, rtol=10e-7):
     return math.fabs(a - b) <= (atol + rtol * math.fabs(b))
+
+
+nanmean = np.nanmean
```

Comment:
we should move these assignments into the except handler so that it is more clear which version you are using.